### PR TITLE
Multiple Web Application House Keeping Items 

### PR DESCRIPTION
**Pull request by mfixstsci**
Addresses #882, #883, & #884

### CODE REVIEW

**Comment by mfixstsci** [comment]
Ready for Review!

**Review by bourque — APPROVED** [review]
Thanks for these updates @mfixstsci, they all look great!

File: style_guide/accessibility_guidelines.md [hunk 1]
```diff
@@ -28,7 +28,6 @@ Every contribution to the web application must include the following features:
     - Ensure the page is still readable and functional when the text size is doubled
     - Ensure users can navigate through page with just the keyboard and not get trapped
     - Ensure you can see where you are when tabbing through page (e.g. buttons and links are highlighted when in focus)
-    - Add a link before the navigation bar to allow users to jump to main content without tabbing through every `<navbar>` item
```
[the remark this answers is not in the document]
Comment (mfixstsci) [reply style_guide/accessibility_guidelines.md:31]: Thank you!

**Comment by bourque** [comment]
The RTD build seems to be stuck, but since this PR is not changing any API docs I am going to assume its fine.

**Pull request merged**